### ISSUE TITLE
HeatEquation2D example for Workshop October 2024 has zeroed bars in the output

### ISSUE THREAD

**Issue by franzpoeschel** [alpaka-group/alpaka-workshop-slides#32]
Running the heatEquation2D example off from eb4487d (oct2024_workshop branch) gives wrong results for me:

```
> srun -A $PROJID --time=1:00 --partition=small-g --cpus-per-task=1 --ntasks=1 --gres=gpu:1 --gpus-per-task=1 ./heatEquation2D 
srun: job 8182155 queued and waiting for resources
srun: job 8182155 has been allocated resources
Using alpaka accelerator: AccGpuHipRt<2,unsigned int>
Execution results incorrect: Max error = 0.745144 (the grid resolution may be too low)
```
This can also be seen in the output images.
The field seems to be initialized correctly at first:
![Bildschirmfoto vom 2024-10-16 14-49-15](https://github.com/user-attachments/assets/22a1f1bc-1b93-490d-bf62-0cf7044295b9)

But at some point between step 9 and 19, someone put black bars into my output:
![Bildschirmfoto vom 2024-10-16 14-49-21](https://github.com/user-attachments/assets/5b5d4905-6a6c-4dab-9794-b0805e5b4f5e)

I did not see this issue when running at https://github.com/franzpoeschel/alpaka/tree/openpmd_output/example/heatEquation2D, based on https://github.com/ikbuibui/alpaka/tree/workshop_example/example/heatEquation2D (but with a newer dev merged).

@chillenzer @ikbuibui

**Comment by ikbuibui** [comment]
Which code stage are you running? It will help me debug this.
Could you also try it with [recap_after_7_AddingSharedMemory](https://github.com/alpaka-group/alpaka-workshop-slides/tree/oct2024_workshop/Day_2/recap_after_7_AddingSharedMemory), and see if it happens there, as that is the final code stage the participants will have

**Comment by ikbuibui** [comment]
Thanks for catching this, was a copy-paste error. Fixed by f7b3ee87b7e1c15ac2228513c147e8709de33e99
